src/components/PostDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Text } from "native-base";
+import { Post } from "../xplat/types/post";
+import { useState, useEffect} from 'react';
+import { Comment } from "../xplat/types/comment";
+import CommentDisplay from "./PostDetailsComment";
+
+const PostDetails = ({post}: {post: Post | undefined}) => {
+    const [comments, setComments] = useState<Comment[] | undefined>();
+
+    useEffect( () => {
+        post?.getComments().then( (data) => setComments(data));
+    }, [post]);
+
+
+    return (
+        <Box flexDir={'column'} margin={2}>
+            {post !== undefined ? 
+                <Box>
+                    <Text alignSelf={'center'}>Comments</Text>
+                    {comments === undefined || comments!.length === 0 ? 
+                        <Box marginTop={2}><Text alignSelf={'center'}>No comments just yet.</Text></Box> 
+                        :
+                        comments?.map( (value, index) => {
+                            return (
+                                <Box key={index} margin={2}>
+                                    <CommentDisplay comment={value}/>
+                                </Box>
+                            )
+                    })
```

Comment:
nit: }) should be tabbed over one more time

Reply:
Very true, this will be fixed after this merge when I push the linter initialization. Don't want to further crowd this PR since it's already hefty